aws-py-oidc-provider-pulumi-cloud/README.md

```diff
@@ -93,6 +93,19 @@ You can configure more granular access control by adding the `sub` claim to the
     ]
 }
 ```
+
+{{< notes type="warning" >}}
```

Comment:
This a Hugo shortcode expression, so it won't render properly outside of our website (e.g., in this GitHub repo). We should adjust it to be regular GitHub-flavored Markdown.

Reply:
Hey @cnunciato,

Doesn't this get funneled into the registry? I think this is why I included it if I remember correctly. What's the best way to highlight this in both places?

Reply:
Sorry for the delayed response! I completely missed this.

It does, but we want it to render properly on GitHub also. Right now, it looks like this:

![image](https://github.com/pulumi/examples/assets/274700/b9f8c923-83b2-48ce-b97f-3cae6be809b9)

Best thing would probably be to use a regular heading and paragraph to help call attention to it. Folks sometimes use blockquotes for callouts as well (`>`), so that's another option. Either would be fine, IMO.